pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -2364,7 +2365,12 @@ public ByteBufPair newMessageAndIntercept(long consumerId, long ledgerId, long e
                 ackSet);
         ByteBufPair res = Commands.serializeCommandMessageWithSize(command, metadataAndPayload);
         try {
-            getBrokerService().getInterceptor().onPulsarCommand(command, this);
+            val brokerInterceptor = getBrokerService().getInterceptor();
```

Comment:
If `getInterceptor()` can return a null, it would also be good to return `Optional<Interceptor>` instead